configuration/scripts/post_start/copyOfflineConcepts.py

```diff
@@ -114,7 +114,7 @@ def execute() :
         insertConceptToOfflineConcepts(concepts[index], uuid, name)
 
 def getTopLevelParents() :
-    query = "select DISTINCT concept_id from concept_name where name = 'All Observation Templates' or name = 'Registration Offline Concepts' or name = 'Diagnosis Set of Sets' or name = 'Visit Diagnoses'"
```

Comment:
@ouiliam here we don't sync the "All Observation Templates" concepts anymore?

Reply:
@rbuisson